packages/blockfrost/src/blockfrostChainHistoryProvider.ts

```diff
@@ -238,7 +239,7 @@ export const blockfrostChainHistoryProvider = (blockfrost: BlockFrostAPI, logger
 
   const transactionsByAddresses: ChainHistoryProvider['transactionsByAddresses'] = async ({
     addresses,
-    sinceBlock
+    blockRange
```

Comment:
Need to also add pagination support for blockfrost
```suggestion
    blockRange,
    pagination
```

Reply:
See 8c1b7b2acec316796833ef6a8a0bc800204b389c commit